other-builds/ndkbuild/hello-neon/app/build.gradle

```diff
@@ -37,10 +37,6 @@ android {
             dimension "abi"
             ndk.abiFilters 'armeabi-v7a'
         }
-        x86 {
```

Comment:
I don't speak gradle, but isn't this removing support for x86* from this sample?

Reply:
this is a battle: like not to build for x86 as the performance number on x86 is slower than the C version of the sample;  it makes great senses for armeabi-v7a though.

Reply:
So? It's a sample. If the NEON emulation performance isn't even good enough for a sample we should just remove it entirely.

Reply:
probably the best action now is: keep the x86 build as-is ( the purpose of the PR is to get rid of the warning of armeabi in r16).
created a bug to look at it deeply:  https://github.com/googlesamples/android-ndk/issues/472